scripts/prepare-swagger-push.sh

```diff
@@ -53,6 +59,9 @@ fi
 # Replace the version placeholder with the new version number.
 perl -pi -e 's/\{OPENAPI_VERSION\}/'$version_number'/' "${SWAGGER_JSON_FILE}"
 
+# Replace the host placeholder with the environment-specific host URL.
```

Comment:
I can't comment on here, but we should do this perl command on line 45 (after the first perl command) as well. The SHA sum will always not match if we don't, and we will increment the version on every commit.

Reply:
Thanks, fixed.

Reply:
Fixed, thanks.